applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/eigensolver_strategy.hpp

```diff
@@ -356,9 +358,9 @@ class EigensolverStrategy
         }
         else
         {
-            SparseSpaceType::Resize(rb, SparseSpaceType::Size1(rStiffnessMatrix));
```

Comment:
I know this is wrong, but we have to discuss what to do with the reference esizes.